includes/PlanFactory.php

```diff
@@ -255,6 +255,7 @@ public static function determine_site_type(): string {
 	 * @return Plan The created plan
 	 */
 	public static function create_plan( string $plan_type, array $custom_plan_data = array() ): Plan {
+		require_once NFD_NEXTSTEPS_DIR . '/includes/Data/Plans/BasePlan.php';
```

Comment:
We should be able to use autoloading to properly load the base in each class that extends it